trl/trainer/grpo_trainer.py

```diff
@@ -585,7 +585,9 @@ def __init__(
 
         # Processing class
         if processing_class is None:
-            processing_class = AutoProcessor.from_pretrained(model.config._name_or_path, padding_side="left")
+            processing_class = AutoProcessor.from_pretrained(
+                model.config._name_or_path, padding_side="left", use_fast=True
```

Comment:
Why do we need to specify `use_fast=True`?

Reply:
No needed actually. Removed